bftengine/src/preprocessor/messages/PreProcessReplyMsg.hpp

```diff
@@ -77,10 +75,8 @@ class PreProcessReplyMsg : public MessageBase {
       NodeIdType senderId, uint16_t clientId, uint16_t reqOffsetInBatch, ReqId reqSeqNum, uint64_t reqRetryId);
   Header* msgBody() const { return ((Header*)msgBody_); }
 
- private:
```

Comment:
I prefer to have separate 'private', 'public', etc. sections for functions and data members.

Reply:
ok. will return.